overlord/hookstate/fdesetup.go

```diff
@@ -19,14 +19,18 @@
 
 package hookstate
 
```

Comment:
this will conflict with #9718

Reply:
I merged it in here now and resolved the conflicts